src/sfma/api.py

```diff
@@ -44,6 +49,11 @@ def __init__(self,
         self.l_linear = l_linear
         self.include_intercept = include_intercept
         self.include_gamma = include_gamma
+        self.pct_trimming = pct_trimming
+        self.constr_grid_num = constr_grid_num
+
+        if pct_trimming > 1.0:
```

Comment:
Also good to check the case less than 0. `if pct_trimming < 0 or pct_trimming > 1`

Reply:
77efaf0